docs/en-US/component/datetime-picker.md

```diff
@@ -77,34 +77,34 @@ datetime-picker/default-time
+| readonly                | whether DatePicker is read only                                                                                | ^[boolean]                                          | —                                                             | false               |
+| disabled                | whether DatePicker is disabled                                                                                 | ^[boolean]                                          | —                                                             | false               |
+| editable                | whether the input is editable                                                                                  | ^[boolean]                                          | —                                                             | true                |
+| clearable               | whether to show clear button                                                                                   | ^[boolean]                                          | —                                                             | true                |
+| size                    | size of Input                                                                                                  | ^[string]                                           | ^[large]/^[default]/^[small]                                           | default             |
+| placeholder             | placeholder in non-range mode                                                                                  | ^[string]                                           | —                                                             | —                   |
+| start-placeholder       | placeholder for the start date in range mode                                                                   | ^[string]                                           | —                                                             | —                   |
+| end-placeholder         | placeholder for the end date in range mode                                                                     | ^[string]                                           | —                                                             | —                   |
+| arrow-control           | whether to pick time using arrow buttons                                                                       | ^[boolean]                                          | —                                                             | false               |
+| type                    | type of the picker                                                                                             | ^[string]                                           | ^[year]/^[month]/^[date]/^[datetime]/ ^[week]/^[datetimerange]/^[daterange]        | date                |
```

Comment:
In cases like this where there are multiple enumeration values, we might be able to use `enum`.